docs/meshcloud.payment-methods.md

```diff
@@ -0,0 +1,96 @@
+
+## The Payment Method Lifecycle: Assigning to Projects
+
+Now that the Partner Admin has created one or more payment methods for a meshCustomer, we are ready to link a payment method to one or more projects. 
+
+First, check if the payment method is correctly created in the meshCustomer. You can do so by navigating to the meshCustomer Account area. In order to do that you need to have Customer Admin rights within the respective customer. On the left-hand side, there should be an item called 'Payment Methods' which you can navigate to. All payment methods that are created and assigned to your current meshCustomer are shown here.
+
+The payment methods that are assigned can be used for both existing projects and new projects.
+
+### Applying a payment method to an existing project
+
+In the Customer Account area, open 'Projects' on the left. Then click on the 'Edit project' button on the right for the project you would like to add a payment method. In the 'Edit Project' screen, you'll see the selected payment methods on the right. As explained at the top of the page, you also have the ability to specify a Substitute Payment Method. 
+
+![Set Payment Method in Project Edit screen](assets/payment_methods/payment_method_selection_project_edit.png)
+
+Select the Active Payment Method of choice, and optionally a Substitute Payment Method if that is desired. Save the project and the payment method(s) will be assigned to the project.
+
+### Applying a payment method to a new project
+
+Create a new project at the top of the meshPanel. Follow the project creation flow until you enter the 'Billing Information' page. 
+
+![Set Payment Method in Project Create screen](assets/payment_methods/payment_method_selection_project_create.png)
+
+As you can see, on this page you have the ability to assign a payment method to the project. Keep in mind that this is mandatory. If you want to specify a substitute payment method, you can do so after creating the project and following the steps above in '**Applying a payment method to an existing project'**. If your meshCustomer has no payment methods, you will be confronted with a warning that looks like this: 
+
+![Missing Payment Method warning](assets/payment_methods/payment_method_missing_in_project_create.png)
+
+After setting a payment method, you can continue the project creation flow and save the new project. You now have a new project with a payment method successfully assigned!
+
+## The Payment Method Lifecycle: Enhance with Metadata
+
+As it is difficult to handle large amounts of payment methods only via their names or identifiers, there is the possibility to provide metadata (in the form of key-value pairs) to the payment methods. This metadata can be provided when creating/editing the payment method in the meshPanel. This is already described in step 8 for '**Creating a payment method via the meshPanel**'.
```

Comment:
in the new api we call it tags. I like metadata! can we reference that metadata on payment methods can be provided by tags?

Reply:
I would first rewrite the payment method to have `meshTags` before I feel comfortable mentioning tags here. I created an idea ticket https://app.clickup.com/t/9mpq7g

Reply:
mh we would have an inconsistency between the meshPaymentMethod endpoint and the UI then.

Reply:
I know they're called tags already there but they are not the same as our meshTags on project/customer/landing zone. I would prefer keeping it like this for now, and ASAP introduce tag schema on payment methods.

Reply:
we will handle that in the cost mgmt initiative